src/Settings.php

```diff
@@ -25,14 +25,24 @@ class Settings {
 	 */
 	public function init() {
 
-		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_assets' ) );
-		add_action( 'in_admin_header', array( $this, 'display_admin_header' ), 100 );
-		add_action( 'admin_menu', array( $this, 'add_settings_page' ) );
-		add_action( 'admin_init', array( $this, 'save_settings' ) );
-		add_action( 'file_upload_types_settings_after_nav_bar', array( $this, 'display_multiple_mimes_support_notice' ) );
-		add_action( 'admin_init', array( $this, 'enable_multiple_mimes_support' ) );
-		add_filter( 'admin_footer_text', array( $this, 'get_admin_footer' ), 1, 2 );
-		add_action( 'admin_print_scripts', array( $this, 'remove_notices' ) );
+		$this->hooks();
+	}
+
+	/**
+	 * Register hooks.
+	 *
+	 * @since {VERSION}
+	 */
+	private function hooks() {
+
+		add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_assets' ] );
```

Comment:
Since we have [this](https://github.com/awesomemotive/wpforms-plugin/wiki/Best-Practices#unused-variables-and-method-arguments) item in the Best Practices, could you add parameters to the callbacks?

Reply:
I am sorry, I am not sure what do you ask. Do you mean to change:
```
add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_assets' ] );
```
into 
```
add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_assets' ], 10, 0 );
```
 (and the rest of `add_action`)?

Reply:
The [admin_enqueue_scripts](https://developer.wordpress.org/reference/hooks/admin_enqueue_scripts/) hook has one argument - `$hook_suffix`.

So, it should be passed into a callback function:
```php
public function enqueue_assets( $hook_suffix ) {
```
The same for other callbacks.

Reply:
Here is what I meant :) - https://github.com/awesomemotive/file-upload-types/pull/46/commits/e3de2eb7f8badc1baa280ab5baec76b5cbf88dda

Reply:
ok, I found admin_menu hook also has parameter (empty string) so I added it as well.